### PR TITLE
Compile supplemental figures for cell type annotation

### LINKED ISSUE #48: Combine panels for supplemental Figure 4
We will need to combine the panels for Supplemental figure 4 into a single file. The file should be letter size - 8.5 x 11.

### PR DESCRIPTION
**Pull request by allyhawkins**
Closes #48 

This PR adds in the illustrator files for the supplemental figures for cell type annotation. These plots are quite large and have a lot of information, so I chose to split this up so S4 is the diagnostic plots for SingleR and CellAssign and S5 is the heatmap comparing submitter annotations to SingleR and CellAssign annotations. 

- For the CellAssign diagnostic plot it was really tricky to change the font size and keep everything aligned because there are some boxes that are linked to the text for the axis labels. I updated the font to be arial, but kept the size at 7. I don't know how we feel about that, but I can just create all new labels in illustrator and that would solve the problem. 
- For the submitter heatmap, I used size 8 for all labels. This heatmap is really big, so the size 6 labels seemed really small.

### CODE REVIEW

**Comment by allyhawkins** [comment]
@jaclyn-taroni These files have been reorganized per https://github.com/AlexsLemonade/scpca-paper-figures/pull/49#issuecomment-1938884084.

**Review by jaclyn-taroni — APPROVED** [review]
👍🏻  one small comment about removing the panel label from S5, but I don't need to see this again.

File: figures/compiled_figures/pngs/figure_s5.png
[the remark this answers is not in the document]
Comment (allyhawkins) [reply figures/compiled_figures/pngs/figure_s5.png:1]: > I don't think we need the panel label for this one.

You mean the `A.` right?

[the remark this answers is not in the document]
Comment (jaclyn-taroni) [reply figures/compiled_figures/pngs/figure_s5.png:1]: Yes, that's correct.

**Pull request merged**